conda/core/index.py

```diff
@@ -171,6 +171,19 @@ def _supplement_index_with_system(index):
         rec = _make_virtual_package('__' + libc_family, libc_version)
         index[rec] = rec
 
+    for flag in context.cpu_flags:
+        if os.getenv("CONDA_OVERRIDE_CPU_FEATURE_{}".format(flag.upper()), "1") == "":
+            continue
+        rec = _make_virtual_package('__cpu_feature_' + flag.lower(), "1.0.0")
```

Comment:
```suggestion
        rec = _make_virtual_package('__cpu_feature_' + flag.lower(), "1")
```
`1` is a valid "version", too and having minor/patch parts does not make much sense for flags.

Reply:
I'd also like us to have an option to disable/remove such flag, e.g., to ensure no feature-dependent packages are used during build time.
(That doesn't have to be addressed in this PR though. I'm generally not too happy with those `OVERRIDE` env vars and will open an issue addressing that.)